lib/lost-center.js

```diff
@@ -1,10 +1,13 @@
 var newBlock = require('./new-block.js');
 
+var lgLogic = require('./_lg-logic');
+
 /**
  * lost-center: Horizontally center a container element and apply padding
  * to it.
  *
- * @param {length} [max-width] - A max-width to assign. Can be any unit.
```

Comment:
@codebysubtract With these edits here, do you have any thoughts on: https://github.com/peterramsing/lost/pull/364